src/main/java/tools/jackson/databind/deser/jdk/JDKKeyDeserializer.java

```diff
@@ -360,13 +360,13 @@ final static class EnumKD extends JDKKeyDeserializer
         protected final AnnotatedMethod _factory;
 
         /**
-         * Alternative to parse enums using {@code toString()} method as the source.
+         * Alternative to parse using {@code toString()} method as the source.
```

Comment:
This does not read well to me. `to parse enums using` makes sense - if there is no object in the clause (`to parse using`) that sounds strange to me. I would suggest not making this edit.